src/Microsoft.ML.Core/Properties/AssemblyInfo.cs

```diff
@@ -13,6 +13,7 @@
 [assembly: InternalsVisibleTo(assemblyName: "Microsoft.ML.InferenceTesting" + PublicKey.TestValue)]
 [assembly: InternalsVisibleTo(assemblyName: "Microsoft.ML.StaticPipelineTesting" + PublicKey.TestValue)]
 [assembly: InternalsVisibleTo(assemblyName: "Microsoft.ML.OnnxTransformerTest" + PublicKey.TestValue)]
+[assembly: InternalsVisibleTo(assemblyName: "Microsoft.ML.AutoML.Tests" + PublicKey.TestValue)]
```

Comment:
Why do the tests need IVT access to these assemblies?

Reply:
Ugh.  For using internal cursor object for a dummy DataView test fixture.  And the tests were all shooting in the air since they were expecting a certain exception for some specific conditions they were trying to test, but actually they all passed simply because the same exception is thrown if the dataset is empty (which it was).  Fixed test validation to be specific, fixed tests that weren't set up correctly so they produce the error they were looking to test.  And removed internalsvisible of course.

Reply:
Hmm not so easy, we had a copy of ArrayDataViewBuilder in AutoML, removing that brings back the issue.